src/simdat.jl

```diff
@@ -20,6 +20,7 @@ Dict(
     :factor2_name => ["F2_level1", "F2_level2", "F2_level3"]
 )
 ```
+The order of columns is typically: :subj, subject-within/between conditions, :item, item-within/between conditions, :dv
```

Comment:
I'm not sure I want to commit to this from an API perspective, so I would recommend removing it.

```suggestion
```